chore(blog): april summary article (#2899)
asyncapi/website: pages/blog/2024-april-summary.md

date: 2024-04-30T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2024-blog-banner/blog-banner-april.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: AsyncAPI Community Manager
excerpt: 'Community update for April'
featured: true
---

We are excited and commend [Specmatic](https://specmatic.in/) for their commitment to supporting open-source projects. They have sponsored [Pavel Bodiachevskii](https://www.linkedin.com/in/pavel-bo/), whose project assists in parsing AsyncAPI specs in Java and converting the parsed specs into executable contracts using the jasyncapi library. This kind of support is truly invaluable to the community and open-source maintainers. Please [check out the latest jasyncapi release](https://github.com/asyncapi/jasyncapi/releases/tag/1.0.0-RC) for more details.

## AsyncAPI Conf on Tour

We are thrilled to announce that AACoT'24 will return to London this year, and [the Call for Speakers is now open](https://conference.asyncapi.com/venue/London). Start preparing your proposals today, and get ready to join us in September!

> On an exciting note, we plan to host an online edition sometime this year. For more context, [please attend the upcoming community meeting](https://github.com/asyncapi/community/issues/1184).

Reviewer: not sure we can say we plan,
it was just me and you having a convo and loved the idea, 

`we have an idea`?

Author: I have faith that why I used plan but yeah let use `idea` instead.😝

Reviewer: sorry, my atheistic soul kicked in 😄

## Spec X Tooling
The [AsyncAPI Optimizer](https://github.com/asyncapi/optimizer) has reached v1.0 thanks to the collaborative efforts of [Viacheslav Turovskyi](https://github.com/aeworxet), who has joined as the new maintainer. For breaking changes, [check out the release notes](https://github.com/asyncapi/optimizer/releases/tag/v1.0.0).

[Aklivity](https://www.aklivity.io/), one of the initiative sponsors, is developing Zilla, a multi-protocol service proxy for Kafka that now supports AsyncAPI v3 for configuration and validation. Please [read the announcement blog](https://www.aklivity.io/post/announcing-openapi-and-asyncapi-support-in-zilla) for more technical details and explore [the Zilla Petstore Kafka API v2 docs](https://tinyurl.com/zilla-petstore-kafka-v2) or [Petstore app demo](https://github.com/aklivity/zilla-demos/blob/main/petstore/README.md).

> In our efforts to improve the contributor experience, we have introduced new help commands: **/ptal**, which pings reviewers to please take a look at the PR, and **/update**, which triggers the AsyncAPI bot to update a PR to the latest upstream. We are grateful to [Priyansh Mehta](https://github.com/Priyansh61), [Khuda Dad Nomani](https://www.linkedin.com/in/khudadadnomani), and [Ashish Padhy](https://www.linkedin.com/in/ashish-padhy3023) for their contributions.

Reviewer: ```suggestion
## Contributor Experience

In our efforts to improve the contributor experience, we have introduced new help commands: **/ptal**, which pings reviewers to please take a look at the PR, and **/update**, which triggers the AsyncAPI bot to update a PR to the latest upstream. We are grateful to [Priyansh Mehta](https://github.com/Priyansh61), [Khuda Dad Nomani](https://www.linkedin.com/in/khudadadnomani), and [Ashish Padhy](https://www.linkedin.com/in/ashish-padhy3023) for their contributions.

Don't worry if you do not remember all these commands, you can always put `/help` command in a pull request and AsyncAPI bot with share with you a list of available commands.
```

---
title: "Community Update: April 2024"
date: 2024-04-25T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2024-blog-banner/blog-banner-april.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: AsyncAPI Community Manager

> excerpt: 'Community update for April'

Reviewer 2: ```suggestion
excerpt: 'April Community update'
```

featured: true
---

> We are excited and commend [Specmatic](https://specmatic.in/) for their commitment to supporting open-source projects. They have sponsored [Pavel Bodiachevskii](https://www.linkedin.com/in/pavel-bo/), whose project assists in parsing AsyncAPI specs in Java and converting the parsed specs into executable contracts using the jasyncapi library. This kind of support is truly invaluable to the community and open-source maintainers. Please [check out the latest jasyncapi release](https://github.com/asyncapi/jasyncapi/releases/tag/1.0.0-RC) for more details.

Reviewer 2: ```suggestion
We are excited and commend [Specmatic](https://specmatic.in/) for their commitment to supporting open-source projects. They have sponsored [Pavel Bodiachevskii](https://www.linkedin.com/in/pavel-bo/), whose project assists in parsing AsyncAPI specs in Java and converting the parsed specs into executable contracts using the `jasyncapi` library. This kind of support is truly invaluable to the community and open-source maintainers. Please [check out the latest `jasyncapi` release](https://github.com/asyncapi/jasyncapi/releases/tag/1.0.0-RC) for more details.
```

[Aklivity](https://www.aklivity.io/), one of the initiative sponsors, is developing Zilla, a multi-protocol service proxy for Kafka that now supports AsyncAPI v3 for configuration and validation. Please [read the announcement blog](https://www.aklivity.io/post/announcing-openapi-and-asyncapi-support-in-zilla) for more technical details and explore [the Zilla Petstore Kafka API v2 docs](https://tinyurl.com/zilla-petstore-kafka-v2) or [Petstore app demo](https://github.com/aklivity/zilla-demos/blob/main/petstore/README.md).

In our efforts to improve the contributor experience, we have introduced new help commands: **/ptal**, which pings reviewers to please take a look at the PR, and **/update**, which triggers the AsyncAPI bot to update a PR to the latest upstream. We are grateful to [Priyansh Mehta](https://github.com/Priyansh61), [Khuda Dad Nomani](https://www.linkedin.com/in/khudadadnomani), and [Ashish Padhy](https://www.linkedin.com/in/ashish-padhy3023) for their contributions.

Don't worry if you don't remember all these commands; you can always use the **/help** command in a pull request, and the AsyncAPI bot will share a list of available commands.

## Community Discussions

> Here are some community discussions that require your attention:

Reviewer 2: ```suggestion
Below are some community discussions that require attention.
```
